java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/SdkOption.java

```diff
@@ -62,4 +66,28 @@ public long getRequestTimeout() {
     public void setRequestTimeout(long requestTimeout) {
         this.requestTimeout = requestTimeout;
     }
+
+    public String getHost() {
+        return host;
+    }
+
+    public void setHost(String host) {
+        this.host = host;
+    }
+
+    public long getPort() {
+        return port;
+    }
+
+    public void setPort(long port) {
+        this.port = port;
+    }
+
+    public boolean isClusterMode() {
+        return isClusterMode;
+    }
+
+    public void setClusterMode(boolean clusterMode) {
```

Comment:
where the option is set ?

Reply:
before use SqlClusterExecutor we should set option, see SQLRouterSmokeTest

Reply:
> where the option is set ?

before use SqlClusterExecutor we should set option, see SQLRouterSmokeTest